Microsoft.Toolkit.Uwp.UI.Controls/FlexPanel/Enums/FlexAlignItems.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See the LICENSE file in the project root
+// for the license information.
+//
+// Author(s):
+//  - Laurent Sansonetti (native Xamarin flex https://github.com/xamarin/flex)
+//  - Stephane Delcroix (.NET port)
+//  - Ben Askren (UWP/Uno port)
+//
+using System;
+using System.ComponentModel;
+using System.Globalization;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Values for <see cref="P:Microsoft.Toolkit.Uwp.UI.Controls.FlexItem.AlignItems" />.
+    /// </summary>
+    [TypeConverter(typeof(FlexAlignItemsTypeConverter))]
+    public enum FlexAlignItems
```

Comment:
Rename to FlexItemAlignment. Does it make sense to have this and the FlexAlignContent enum? Can we just use one? Maybe some values are not valid for alignment of items?